src/components/organisms/Room/Participant.tsx

```diff
@@ -1,12 +1,18 @@
-import React, { useEffect, useMemo, useRef, useState } from "react";
```

Comment:
in this file: let's use named export instead of default one.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2221/commits/38cc1cb1569924f337d9cc44a131f843e8658c1f